core/ipc/rpc/jms-impl/src/main/resources/OSGI-INF/blueprint/blueprint-rpc-server.xml

```diff
@@ -48,4 +48,12 @@
     <reference-list id="rpcModulesRef" interface="org.opennms.core.rpc.api.RpcModule" availability="optional">
         <reference-listener bind-method="bind" unbind-method="unbind" ref="jmsRpcServerRouteManager"/>
     </reference-list>
+
+    <!-- JMS HealthCheck -->
+    <service interface="org.opennms.core.health.api.HealthCheck">
+        <bean class="org.opennms.distributed.jms.impl.JmsConnectionHealthCheck" >
+            <argument ref="blueprintBundleContext"/>
+            <argument value="RPC" />
```

Comment:
I don't think the differentiation is required as we only check if a connection to the jms is possible in general. 
This looks like a workaround to me to hide the fact, that we are exposing the same health-check twice.

Reply:
Note that Kafka/ActiveMQ can be enabled either for Sink/RPC.  When user does health:check they should know exactly what they are doing and which configuration is wrong.  So this is very much needed